projects/test_project/rules.mk

```diff
@@ -1,8 +1,6 @@
-# Specify the device family to build for
-DEVICE_FAMILY := stm32f0xx
-
 # Specify the main file (relative to this file if not in this folder)
 MAIN := main.c
 
 # Specify the device library you want to include
-DEVICE_LIBRARY := libstm32f0.a
+DEPS := libraries/stm32f0xx/stm32f0 \
```

Comment:
I can for sure, but if you use a path relative to the Makefile it allows you to also create mini-libraries to depend on within your project ie: if you need a custom data-structure or stand alone component you can create it as a library for the project rather than a library in the libraries folder. This is a nice option IMO but if you want to require all libraries to go in libraries then this is valid.